index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <!--NAV VAR-->
+    <header>
+        <nav>
+            <p>My logo</p>
```

Comment:
- [ ] Please use an `a` tag to wrap the logo with an `href` attribute pointing to the index HTML file. This is best for user experience so that clicking on the logo will take you to the home page.